scripts/cluster-redirect.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I think that this file + `scripts/kubernetes/resources.sh` and `scripts/openshift/resources.sh` should be combined into a single file at `scripts/lib/cluster.sh`. Looking something like:

```
if [ -n "${KUBERNETES}" ]; then
    echo "Kubernetes Cluster"
    function asb::cluster::deployments {
        ...
    }
    ...
else
    echo "OpenShift Cluster"
    function asb::cluster::deployments {
        ...
    }
    ...
fi
```

Reply:
I never really liked this cluster redirect script. What I'm going to do is add it to the top of the prep_local_dev_env.sh.

Similar to deploy.sh, I like having the abstraction of a directory for $CLUSTER files. I think it makes it easy to separate $CLUSTER scripts without overloading a single file.

Reply:
@djzager -1 to the combining the files. while it does duplicate somethings it is way easier to see. Does this change kubernetes? open scripts/kubernetes files. Does it affect openshift? open scripts/openshift files. if they are combined you have to be more cognizant not to break the other worrying about the conditionals.

Reply:
@jmrodri fair point.

My general feeling is that we are turning a single file into 4 to prepare the local environment, when there are only subtle differences between how to prepare the environment in k8s vs openshift and that strikes me as overkill.

Having said that, @rthallisey, delete `scripts/openshift/resources.shs` and I'll change my vote.

Reply:
@djzager we do end up with 4, but prep_local_devel_env.sh is already hard to read and I think this does minimal damage to it's readability.  I do think there is a bunch of overlap here and a lot of potential for improvements.  This patch is mainly for functionality without changing how the script reads.  If you want to improve it, there is definitely an opportunity to and I would encourage it! But, it wasn't my intention with this patch. 

I'll remove the extra file.